src/main/java/be/syntra/devshop/DevshopBack/security/entities/CustomUser.java

```diff
@@ -0,0 +1,29 @@
+package be.syntra.devshop.DevshopBack.security.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.User;
+
+import java.util.Collection;
+
+@Getter
+@Setter
+public class CustomUser extends User {
+
+    private Long userID;
+
+    /*
+     * This is a custom Object upon which we set our actual security
+     * it seperates our User business logic from the little that's needed for security
```

Comment:
rename class so some of the comment lines can be deleted.

Reply:
done